compiler/tests/org/jetbrains/kotlin/generators/tests/GenerateInRangeExpressionTestData.kt

```diff
@@ -28,17 +17,18 @@ object GenerateInRangeExpressionTestData {
     private val PREAMBLE_MESSAGE = "Auto-generated by ${GenerateInRangeExpressionTestData::class.java.simpleName}. Do not edit!"
 
     private fun generateMatrixTestCase(
-            fileName: String,
-            rangeExpressions: List<String>,
-            elementExpressions: List<String>,
-            header: String = ""
+        fileName: String,
+        rangeExpressions: List<String>,
+        elementExpressions: List<String>,
+        header: String = ""
     ) {
         PrintWriter(File(GENERATED_DIR, fileName)).use {
             it.generateTestCaseBody(header, rangeExpressions, elementExpressions)
         }
     }
 
     private fun PrintWriter.generateTestCaseBody(header: String, rangeExpressions: List<String>, elementExpressions: List<String>) {
+        println("// KJS_WITH_FULL_RUNTIME")
```

Comment:
Why this change is required? Are affected tests was broken?

Reply:
Or this line was added manually for non affected?

Reply:
It was added manually before for the "in range" tests, instead of through this test generator. It was only added for the tests that needed it; there are 6 affected tests. Now we want this generator to be called during test generation. It was cleaner to add the directive for all 14 tests generated by this generator—whether it was needed or not for the test—rather than adding it only for the 6 tests that needed it.